src/components/month/calendarMonthView.component.ts

```diff
@@ -54,7 +54,7 @@ import { CalendarEventTimesChangedEvent } from '../../interfaces/calendarEventTi
         <div *ngFor="let rowIndex of view.rowOffsets">
           <div class="cal-cell-row">
             <mwl-calendar-month-cell
-              *ngFor="let day of view.days | slice : rowIndex : rowIndex + 7"
+              *ngFor="let day of getDays() | slice : rowIndex : rowIndex + (count())"
```

Comment:
This will run on every change detection cycle which inefficient, these function call results should be ran once on the ngOnChanges when the `excludeDays` input changes